client/settings/digital-wallets/test/digital-wallets.test.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+/**
+ * Internal dependencies
+ */
+import DigitalWallets from '..';
+
+describe( 'DigitalWallets', () => {
+	it( 'toggles the payment method locations status', async () => {
+		render( <DigitalWallets /> );
+
+		const [
+			,
+			checkoutCheckbox,
+			productPageCheckbox,
+			cartCheckbox,
+		] = screen.getAllByRole( 'checkbox' );
+
+		// all "locations" checkboes are disabled and unchecked.
+		expect( checkoutCheckbox.disabled ).toBeTruthy();
+		expect( checkoutCheckbox.checked ).toBeFalsy();
+		expect( productPageCheckbox.disabled ).toBeTruthy();
+		expect( productPageCheckbox.checked ).toBeFalsy();
+		expect( cartCheckbox.disabled ).toBeTruthy();
+		expect( cartCheckbox.checked ).toBeFalsy();
+
+		userEvent.click(
+			screen.getByText(
+				'Enable digital wallets & express payment methods'
+			)
+		);
+
+		// only the checkout checkbox is checked by default, once the feature is enabled.
```

Comment:
```suggestion
		// all checkboxes are checked by default, once the feature is enabled.
```

Reply:
lol there's been a bit of back and forth on how this should have been working, I thought I deleted all the old relics :D thanks!